airflow/config_templates/default_airflow.cfg

```diff
@@ -122,6 +122,12 @@ security =
 # values at runtime)
 unit_test_mode = False
 
+# Logging configuration path
+logging_config_path = airflow.logging.airflow_logging_config.AIRFLOW_LOGGING_CONFIG
+
+# Name of handler to read task instance logs
+task_log_reader = airflow.task
```

Comment:
this isn't the handler, but rather the name of the handler, i.e. `airflow.task` vs `airflow.utils.log.TaskFileHandler`, that should be clear from the config. It might be better to include this with the log config, e.g. as part of the dict. This would place all log configuration in one place.